RadixWallet/Core/FeaturePrelude/Loadable.swift

```diff
@@ -307,16 +307,29 @@ extension Loadable {
     public mutating func refresh(
         from other: Loadable<Value>,
         valueChangeMap: (_ old: Value, _ new: Value) -> Value = { _, new in new }
-    ) {
+    ) where Value: Equatable {
```

Comment:
Updated this function to be more explicit of when updates of the current values occur or not.